main/res/layout/checkbox_item.xml

```diff
@@ -47,6 +50,7 @@
         android:id="@+id/item_checkbox"
         android:layout_width="30dp"
         android:layout_height="wrap_content"
+        android:minHeight="32dip"
```

Comment:
Isn't that covered by the main theme defining a `checkboxStyle`, which defines a `minHeight`?

Reply:
> Isn't that covered by the main theme defining a `checkboxStyle`, which defines a `minHeight`?

You're right, that was an unwanted leftover from my "experiments". I removed it, thanks for the review!